modules/appeals_api/app/swagger/appealable_issues/v0/swagger_dev.json

```diff
@@ -340,35 +340,45 @@
       },
       "productionOauth": {
         "type": "oauth2",
-        "description": "This API uses OAuth 2 with the authorization code grant flow. [More info](https://developer.va.gov/explore/authorization?api=claims)",
+        "description": "The authentication model for the Appealable Issues API uses OAuth 2.0/OpenID Connect. The following authorization models are supported: [Authorization code flow](https://dev-developer.va.gov/explore/api/appealable-issues/authorization-code) and [Client Credentials Grant (CCG)](https://dev-developer.va.gov/explore/api/appealable-issues/client-credentials).",
```

Comment:
I used dev developer portal URLs here to keep consistent with the rest of the content in the dev and staging docs.